ppxlib.opam

```diff
@@ -21,7 +21,8 @@ depends: [
   "ppx_derivers"            {>= "1.0"}
   "sexplib0"
   "stdlib-shims"
-  "ocamlfind"               {with-test}
+  "ocamlfind"
```

Comment:
I'd be curious to understand why ocaml-ci is complaining about this. I'd much prefer not to introduce a dependency on ocamlfind here!